cms/models.py

```diff
@@ -29,6 +29,18 @@ class Meta:
         blank=True,
         help_text="A short description indicating how long it takes to complete (e.g. '4 weeks')",
     )
+    time_commitment = models.CharField(
+        max_length=100,
+        null=True,
+        blank=True,
+        help_text="A short description indicating about the time commitments",
+    )
+    format = models.CharField(
```

Comment:
can we change this, as `format` is a keyword in python. We should avoid using it as field name.

Reply:
sure.